.github/workflows/mdspan-version-check.yml

```diff
@@ -1,7 +1,16 @@
 name: mdspan version check
 
 on:
-  workflow_call:
+  push:
+    paths:
+      - '.github/workflows/mdspan-version-check.yml'
+      - 'tpls/mdspan/**'
+    branches:
+      - develop
+  pull_request:
```

Comment:
Should we be testing PRs only?
Not feeling too strongly on this but that would remove that duplicated filter block

Reply:
Correct